docs/CHANGELOG.rst

```diff
@@ -27,6 +27,10 @@ Added
   ``alignment-star`` process
 - Add mappings from ENSEMBL or NCBI to UCSC chromosome names to
   ``resolwebio/rnaseq:3.5.0`` docker image
+- Add ``bigwig_chroms_to_ucsc.py`` to ``bamtobigwig.sh`` to change chromosome
```

Comment:
This is just as suggestion, but i would rephrase this to:

Change chromosome names from ENSEMBL / NCBI to UCSC (example: "1" to "chr1") in BigWig files. The purpose of this is to enable viewing BigWig files in UCSC genome browsers for files aligned with ENSEBML or NCBI genome. This change is done by adding script bigwig_chroms_to_ucsc.py to bamtobigwig.sh script.

Reply:
Nice :D